app/models/game.rb

```diff
@@ -1,4 +1,63 @@
 class Game < ActiveRecord::Base
   belongs_to :users
```

Comment:
Should we try this... 

class Game < ActiveRecord::Base
  belongs_to :player_white, class: 'User'
  belongs_to :player_black, class: 'User'

Because player_white and player_black assumes in schema.rb that games table has a column named player_white_id